localenv/README.md

```diff
@@ -109,6 +109,35 @@ The secondary Happy Life Bank docker compose file (`./happy-life-bank/docker-com
 data stores created by the primary Rafiki instance so it can't be run by itself.
 The `pnpm localenv:compose up` command starts both the primary instance and the secondary.
 
+#### Autopeering
+
+If you want to start the local env and peer it automatically to rafiki.money, you can run the following commands:
+
+```
+pnpm localenv:autopeer:start
+
+// OR to start with Postgres db
+pnpm localenv:autopeer:start:psql
+```
+
+Your local cloud nine rafiki instance will be peered automatically in this case with https://rafiki.money instance.
+The required services will be exposed externally using [ngrok](https://www.npmjs.com/package/ngrok) and [tunnelmole](https://www.npmjs.com/package/tunnelmole) packages.
+Ngrok is used to expose open-payments api and auth api. These apis require a https protocol for correctly validating the requests signatures and ngrok will add X-Forwarded-Proto to secure requests that helps koa to identify requests as secure.
+Tunnelmole package is used for exposing externally ilp connector.
+
+There is no need for ngrok token in case you are planning to try out only the API requests.
+In case you are looking to test the open payments example with the interaction flow, then the ngrok token should be provided in `~/cloud-nine-wallet/.env` file
+as `NGROK_TOKEN={YOUR TOKEN}`. Ngrok does not serve html without providing the auth token.
```

Comment:
@AlexLakatos can we get an ngrok account and use that token globally? Otherwise the developer experience isn't great.

Reply:
I thought we weren't using ngrok. What happened to the other options?

Reply:
I was using tunnelmole, but the issue was that the requests are redirected as `http` and the signature is not validated as it is expecting `https`.
Koa application has an option to trust the proxy and then it will check the presence of `X-Forwarded-Proto` header. This header is added by ngrok, but not by tunnelmole - this is why ngrok was used for the services that validate signature requests.

The issue with ngrok is that it works fine without a token on api request, but it does not serve `html`. So for testing the interaction part on open payments it requires a token (a free one works) to be provided.

Reply:
@sabineschaller @mkurapov does these changes make sense to you?
https://github.com/interledger/rafiki/commit/ae604cc691323ea4d89f69e06b755f2c02655c39

Reply:
Would that mean we get away without using ngrok?

Reply:
yes, it will allow to use only tunnelmole

Reply:
Let's do that but check `Config.env === 'development'` rather than `Config.env === 'testing'`, or just both.